scripts/deploy-backend.sh

```diff
@@ -62,11 +62,13 @@ set_cf_envs()
   for var_name in ${var_list[@]}; do
     # Intentionally not setting variable if empty
     if [[ -z "${!var_name}" ]]; then
-        echo "WARNING: Empty value for $var_name"
+        echo "WARNING: Empty value for $var_name. It will now be unset."
+        cf_cmd="cf unset-env $CGAPPNAME_BACKEND $var_name ${!var_name}"
```

Comment:
This will remove stale variables.  I don't think this is actually an important fix since this can also be done manually.